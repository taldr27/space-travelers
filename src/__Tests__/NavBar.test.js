import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../components/NavBar';

test('Render Navbar', () => {
  const renderNavbar = renderer.create(<BrowserRouter><Nav /></BrowserRouter>).toJSON();
  expect(renderNavbar).toMatchSnapshot();
});
