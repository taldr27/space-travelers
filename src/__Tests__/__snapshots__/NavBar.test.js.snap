// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Navbar 1`] = `
<header
  className="header"
>
  <div
    className="logo-title"
  >
    <img
      alt="planet-icon"
      className="logo-img"
      src="planet.png"
    />
    <h1>
      Space Traveler's Hub
    </h1>
  </div>
  <nav>
    <ul
      className="pages"
    >
      <li
        className="child1"
        id={1}
      >
        <a
          aria-current="page"
          className="navBar-link selected"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li
        className="child2"
        id={2}
      >
        <a
          className="navBar-link"
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li
        className="child3"
        id={3}
      >
        <a
          className="navBar-link"
          href="/profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
